example/src/main/deploy/openshift/enmasse_deploy.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
This seems to be a verbatim copy of the `openshift_deploy.sh` script. This approach will make it harder to maintain the scripts because we need to keep them in sync manually. Isn't there a way to re-use the deploy steps in both scripts, e.g. by moving the deploy commands to `common.sh` as well?

Reply:
Yeah, that's a good idea. We can do that in the next iteration of improvements